### PR TITLE
made index page for backend

### PR DESCRIPTION
**Pull request by maitrakhatri**
# things added
- Homepage
- Login
- Register

# things to check
- consistent UI
- Code readability
- Components 

# Screens
image

### CODE REVIEW

File: backend/index.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<?php
+  session_start();
+?>
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
Comment (maitrakhatri) [inline backend/index.php:9]: add meta tag for author, title and desc

File: backend/index.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  </nav>
+
+
+
+
+
+
+
+
+
```
Comment (maitrakhatri) [inline backend/index.php:58]: delete extra space

File: backend/index.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+<!-- FOOTER -->
+<footer>
+  <div class="p-3 mb-2 bg-dark text-white">  
+  <div class="text-monospace">
+  <p class="text-center">Created by</p>
+
+  <ul class="nav justify-content-center">
+  <li class="nav-item">
+    <a class="nav-link" href="https://github.com/hirvaDhandhukia">Hirva</a>
+  </li>
```
Comment (maitrakhatri) [inline backend/index.php:96]: correct intentation

File: backend/index.php [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+    <!-- Option 1: jQuery and Bootstrap Bundle (includes Popper) -->
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns" crossorigin="anonymous"></script>
+
+    <!-- Option 2: Separate Popper and Bootstrap JS -->
+    <!--
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.min.js" integrity="sha384-+YQ4JLhjyBLPDQt//I+STsc9iw4uQqACwlvpslubQzn4u2UU2UFM80nGisd026JF" crossorigin="anonymous"></script>
```
Comment (hirvaDhandhukia) [inline backend/index.php:127]: delete things that are not needed

**Review by maitrakhatri — COMMENTED** [review]
Do all the mentioned changes

**Review by hirvaDhandhukia — CHANGES_REQUESTED** [review]
correct changes please

**Review by hirvaDhandhukia — APPROVED** [review]

**Pull request closed without merging**